src/components/calcite-radio-group/calcite-radio-group.e2e.ts

```diff
@@ -269,4 +269,36 @@ describe("calcite-radio-group", () => {
     expect(child2).toEqualAttribute("scale", "m");
     expect(child3).toEqualAttribute("scale", "m");
   });
+
+  describe("setFocus()", () => {
+    it("focuses the first item if there is no selection", async () => {
+      const page = await newE2EPage({
+        html: `<calcite-radio-group>
+          <calcite-radio-group-item id="child-1" value="1">one</calcite-radio-group-item>
+          <calcite-radio-group-item id="child-2" value="2">two</calcite-radio-group-item>
+          <calcite-radio-group-item id="child-3" value="3">three</calcite-radio-group-item>
+        </calcite-radio-group>`
+      });
+
+      const element = await page.find("calcite-radio-group");
+      await element.callMethod("setFocus");
+
+      expect(await page.evaluate(() => document.activeElement.id)).toEqual("child-1");
+    });
+
+    it("focuses the selected item", async () => {
+      const page = await newE2EPage({
+        html: `<calcite-radio-group>
+          <calcite-radio-group-item id="child-1" value="1">one</calcite-radio-group-item>
+          <calcite-radio-group-item id="child-2" value="2">two</calcite-radio-group-item>
+          <calcite-radio-group-item id="child-3" value="3" checked>three</calcite-radio-group-item>
+        </calcite-radio-group>`
+      });
+
+      const element = await page.find("calcite-radio-group");
+      await element.callMethod("setFocus");
+
+      expect(await page.evaluate(() => document.activeElement.id)).toEqual("child-3");
```

Comment:
I'm not sure document.activeElement.id works here because document.activeElement is undefined

Reply:
Nevermind... it returned undefined and the test failed the first time I ran it, passes now 🤷‍♂

Reply:
Weird. Out of curiosity, how did you run the tests?

Reply:
`npm test calcite-radio-group`

Reply:
Awesome! I didn't know we could specify which suite to run. 

![giphy](https://user-images.githubusercontent.com/197440/76129768-20162000-5fbd-11ea-9fae-3da33cf24d95.gif)

Side note, I haven't been able to repro the failure. I'll keep an eye out. 👁